### PR TITLE
docs: add build instructions

### PR DESCRIPTION
**Pull request by paulbutcher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+export default {
+```
+
+Then:
+
+```
+npm ci 
+npm run build
+serve -s build
+```
```
Comment (magnus-madsen) [inline README.md:25]: Are these commands different from `npm start`?

**Comment by magnus-madsen** [comment]
We could also add a comment to `src/Data.js` that this file should be obtained by running the compiler.

**Comment by paulbutcher** [comment]
I've added the comment to `src/Data.js` as suggested and changed the instructions to reference `npm start` (I was cribbing from the github workflow, and didn't work out that `npm start` would do the trick until afterwards).

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**